bases/esm.json

```diff
@@ -4,6 +4,7 @@
 
   "compilerOptions": {
     "module": "es2022",
+    "moduleResolution": "bundler",
```

Comment:
I strongly object to this base existing at all. It’s a trap that might prevent a user from finding the config they should be using for Node or Vite or whatever.

Reply:
I delisted it in the latest commit.

Reply:
Is the suggested approach to now use base/node18 and then set moduleResolution manually in tsconfig.json to bundler?